lizmap/modules/admin/classes/listProjectDatasource.class.php

```diff
@@ -22,7 +22,7 @@ public function getData($form)
         $criteria = $form->getData($this->criteriaFrom[0]);
         if ($criteria && array_key_exists($criteria, $this->data)) {
             $rep = lizmap::getRepository($criteria);
-            $projects = $rep->getProjects();
+            $projects = $rep->getProjects(lizmap::getAppContext(), lizmap::getServices());
```

Comment:
why these parameters? the repository object should already have these references..